Error when running `npm run build`

`npm run dev` works as expected, with no errors.
But when I try running `npm run build` I get errors.

Steps to reproduce:
* `npm i`
* `npm run build`

Error:
```

> vite-lit-element-ts-sass@0.0.0 build /workspace/vite-lit-element-ts-sass
> tsc && vite build

node_modules/vite/client.d.ts:89:18 - error TS2300: Duplicate identifier 'css'.

89   export default css
                    ~~~

  typings/scss.d.ts:4:18
    4   export default styles;
                       ~~~~~~
    'css' was also declared here.

typings/scss.d.ts:4:18 - error TS2300: Duplicate identifier 'css'.

4   export default styles;
                   ~~~~~~

  node_modules/vite/client.d.ts:89:18
    89   export default css
                        ~~~
    'css' was also declared here.


Found 2 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! vite-lit-element-ts-sass@0.0.0 build: `tsc && vite build`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the vite-lit-element-ts-sass@0.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/gitpod/.npm/_logs/2021-05-05T04_15_50_631Z-debug.log
```

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -15,7 +15,8 @@
     "moduleResolution": "node",
     "allowSyntheticDefaultImports": true,
     "experimentalDecorators": true,
-    "forceConsistentCasingInFileNames": true
+    "forceConsistentCasingInFileNames": true,
+    "skipLibCheck": true
   },
   "include": ["src/**/*.ts","typings/*.d.ts"],
   "exclude": []

